src/services/service.ts

```diff
@@ -1,6 +1,8 @@
 // TODO
-// Every service needs to have a 'help' text 
+// Every service needs to have a 'help' text
 interface IService {
-    buildMessage(): Promise<string>;
-    helpMessage: string;
-}
\ No newline at end of file
+  buildMessage(): Promise<string>;
+  processComponents(args: string[]): void;
+  message: string;
```

Comment:
What benefit does processComponents and message give? you never use any of this new functionality on IService. The key thing here is on IService the interface